beacon-chain/core/peerdas/peer_sampling.go

```diff
@@ -0,0 +1,56 @@
+package peerdas
+
+import (
+	"math/big"
+
+	"github.com/OffchainLabs/prysm/v6/config/params"
+)
+
+// ExtendedSampleCount computes, for a given number of samples per slot and allowed failures the
```

Comment:
Should we wait to merge these and focus on custody sampling first?

Reply:
The whole peer sampling code is currently not used at all.
However, it is already implemented and well unit tested, so there is no harm to merge it as is.
Note I'm not strongly against not to merge it right now. (No strong opinion about it.)

Reply:
I'm not against merging it, but I probably won't review it here because this PR is already big enough and already contains a lot of critical logic to review. We can either merge it to develop and hope someone who uses it later will review it, or merge it later in another `peerDAS` to `develop` PR. Up to you!

Reply:
Removed in 254312058220fc66a3bf1167a1451782131af28b.

Peer sampling is not lost, since still living in the `peerDAS` branch.
For now, let's not merge it into `develop` and postpone the decision to later.